product_docs/docs/pgd/5/rel_notes/pgd_5.3.0_rel_notes.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "EDB Postgres Distributed 5.3.0 release notes"
+navTitle: "Version 5.3.0"
+---
+
+Released: 14 Nov 2023
+
+EDB Postgres Distributed version 5.3.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.3.
+!!!
+
+## Highlights of EDB Postgres Distributed 5.3.0
+
+* Postgres 16 and EPAS 16 supported
+
+## Compatibility
+
+PGD 5.3 is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8, and later.
```

Comment:
This is no longer sufficient a statement. I'll take care of updating this statement.